chpl/chpl-api/e2e/collections/search-controller.postman_collection.json

```diff
@@ -237,6 +257,71 @@
 				"description": "POST /search/beta - filters other than compliance filters - should return 200 status"
 			},
 			"response": []
+		},
+		{
+			"name": "GET /search/beta -using IS_ELIGIBLE should return 400 status",
```

Comment:
How is this test different than the one that was removed expect that was POST and this is GET but both were written originally to make a request with removed parameter and expect bad response and this one still does that? I was suggesting to keep the removed check to expect valid parameters in response - 
 pm.expect(pm.response.text()).to.include(\"Values must be one of HAS_PLANS_URL,HAS_RESULTS_URL,NO_PLANS_URL,NO_RESULTS_URL\");\r", 
 remove check for 'expecting bad response' and change the test to say something like GET /search/beta endpoint returns relevant/expected parameters in response...

Reply:
Yes, this test is different from the other because for GET request passing removed parameters gives 400 status and for POST with removed parameter , the expected status is 500 which is the one I removed . I would like to leave the GET request with removed parameter test as is and I can add another test for POST with valid parameters